pkg/controller/atlasproject/team_reconciler.go

```diff
@@ -7,6 +7,8 @@ import (
 	"net/http"
 	"sync"
 
+	"go.mongodb.org/atlas-sdk/v20231115008/admin"
```

Comment:
If moving to the translation layer, the references to the SDK disappear from here. Instead the reconcile code knows only the translation layer and such translation layer is the only one concerned with the SDK calls.

Reply:
As mentioned in my reply to your previous comment, this PR only concerns converting the API to the new SDK and translation layer will be implemented in a following PR